js/app/cookies-banner.js

```diff
@@ -12,13 +15,27 @@ function submitCookieForm(e) {
     $('.js-accept-cookies').prop('disabled')
         .addClass("btn--primary-disabled");
 
-    document.cookie = "cookies_preferences_set=true; max-age=" + oneYearInSeconds + ";";
-    document.cookie = "cookies_policy=%7B%22essential%22%3Atrue%2C%22usage%22%3Atrue%7D; max-age=" + oneYearInSeconds + ";";
+    document.cookie = "cookies_preferences_set=true; max-age=" + oneYearInSeconds + ";" + "domain=" + cookiesDomain + ";";
+    document.cookie = "cookies_policy=%7B%22essential%22%3Atrue%2C%22usage%22%3Atrue%7D; max-age=" + oneYearInSeconds + ";" + "domain=" + cookiesDomain + ";";
```

Comment:
As discussed 
Like with the way the other cookie is set, I think it is easier to read when values being set to tear them out into their own variables, especially when we have URL encoded strings (not holding back an approval for this mind)